neurodsp/sim/cycles.py

```diff
@@ -204,6 +204,109 @@ def sim_gaussian_cycle(n_seconds, fs, std):
     return cycle
 
 
+def sim_gaussian_asym_cycle(n_seconds, fs, beta, scale=2, shift=1):
+    """Simulate an asymmetrical guassian cycle.
+
+    Parameters
+    ----------
+    n_seconds : float
+        Length of cycle window in seconds.
+    fs : float
+        Sampling frequency of the cycle simulation.
+    beta : float
+        Exponent controlling the amplitude asymmetry of peaks relative to the troughs.
+
+        - `beta=0` : zeros
+        - `beta=.5`: wide peaks, narrow troughs
+        - `beta=1.`: symetrical peaks and troughs (sine wave)
+        - `beta=5.`: wide troughs, narrow peaks
+
+    scale : float, optional, default: 2
+        Rescales the amplitude of the signal.
+    shift : float, optional, default: 1
+        Translate the signal along the y-axis.
+
+    Returns
+    -------
+    cycle : 1d array
+        Simulated asymmetrical gaussian cycle.
+
+    Notes
+    -----
+    If shift or scale keyword arguments are given, default signal normalization will be bypassed.
```

Comment:
this note seems somewhat misplaced, because using this function directly, none of our cycles are normalized. This only applies to simulating this cycle with `sim_oscillation`. We probably want to note that _somewhere_, but I don't think here.

Reply:
I can move it to `sim_oscillation` and `sim_bursty_oscillation`?

Reply:
I moved this to a note in the normalize decorator.